meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.path

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Config.json path watch
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Path]
+PathChanged=/mnt/boot/config.json
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
It's been a couple months since I wrote this and I honestly just don't remember any more, I guess it could be multi user target instead as the requires/after should still keep the ordering the same